docs/README-dev.md

```diff
@@ -7,3 +7,19 @@ This project uses [Sphinx Theme](https://sphinx-theme.scylladb.com/) , but we ha
 To build the documentation locally, run `make setup` first to install its dependencies.
 
 Then, you can build and preview the documentation by following the steps outlined in the [Quickstart guide](https://sphinx-theme.scylladb.com/stable/getting-started/quickstart.html).
+
+## Local preview with Podman
+
+Here is an example how you can start quickly using containers, similarly to how our CI runs it
+
+```bash
+$ podman run -it --pull=Always --rm -v="$( pwd )/docs:/go/$( go list -m )/docs:Z" --workdir="/go/$( go list -m )/docs" -p 5500:5500 quay.io/scylladb/scylla-operator-images:poetry-1.5 bash -euExo pipefail -O inherit_errexit -c 'poetry install && make multiversionpreview'
```

Comment:
Let's delete the dollar sign so that it's easier to copy

Reply:
Shouldn't that say that you need to clone the repository anyway? Since it uses the makefile?

Reply:
> Let's delete the dollar sign so that it's easier to copy

ok, guess I tend to be consistent about bash snippets and in the others, this differentiates the command from it's output.

> Shouldn't that say that you need to clone the repository anyway?

I'd be hesitant to add something that obvious, given it also mounts a docs subdir

Reply:
added a sentence about being at repository root that should put this at ease

Reply:
> I'd be hesitant to add something that obvious, given it also mounts a docs subdir

Ok, it's for developers so I guess we can expect it to be obvious